ocs_ci/framework/__init__.py

```diff
@@ -375,6 +375,28 @@ def is_consumer_exist(self):
         cluster_types = [cluster.ENV_DATA["cluster_type"] for cluster in self.clusters]
         return "consumer" in cluster_types
 
+    def is_hci_client_exist(self):
```

Comment:
I recommend to rename `is_hci_client_exist` and `is_hci_provider_exist` functions to  `hci_client_exists` and `hci_provider_exists` to improve readability - https://stackoverflow.com/questions/1566745/boolean-method-naming-readability